third_party/terraform/website/docs/r/google_iap_tunnel_instance_iam.markdown

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
For this to show up on the docsite, we'll also need to add it to the sidebar, similar to: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/website-compiled/google.erb#L273-L280